packages/dds/matrix/src/matrix.ts

```diff
@@ -568,6 +640,12 @@ export class SharedMatrix<T = any>
 							colsRefSeq,
 						);
 					}
+				} else {
+					this.emit(
```

Comment:
I feel like this could be incorrect. Looking at the "happy" path above, we might not be able to locate a row and/or column as they might be already be deleted. In such case, I expect us not to raise conflict event, as to some extent these changes went to void (and client already knows to deal with them by noticing that row/col was deleted).

The only minus of current flow I see (which exists before your changes) - if the other user will undo delete of row/column, then changes from this client are not going to make it. I'm not sure if it's a serious issue or not, but it's somewhat independent from proposed changes.

Reply:
This will only be executed if that cell is not yet deleted otherwise it will enter into above if logic.

Reply:
Rebase pos handles that scenario properly as mentioned in the other comment. So, we are good here.